TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -51,13 +51,13 @@ class RouteDetailContentViewController: UIViewController, GMSMapViewDelegate, CL
     /** Initalize RouteDetailViewController. Be sure to send a valid route, otherwise
      * dummy data will be used. The directions parameter have logical assumptions,
      * such as ArriveDirection always comes after DepartDirection. */
-    init(route: Route, currentLocation: CLLocationCoordinate2D? = nil) {
```

Comment:
Are you sure you can get rid of this? I tried not having explicitly starting nil as a default and it complained

Reply:
yeah just instead of `RouteDetailViewController(route: route)` you would do smtg like `RouteDetailViewController(route: route, currentLocation: nil)`. I thought I fixed that when I changed the initializer but maybe I missed a few

Reply:
So you're suggesting to not have default parameters when nil is involved?

Reply:
Yeah cause that's the way other library functions do it. Like for completion handlers you just put in nil instead of having the default argument of a completion handler be nil.

Reply:
Ah, ok, I’ve done it on a few places. Will remember